dooit/utils/config.py

```diff
@@ -1,8 +1,9 @@
 import yaml
 from pathlib import Path
+from os import environ
 
 HOME = Path.home()
-XDG_CONFIG = HOME / ".config"
+XDG_CONFIG = Path(environ["XDG_CONFIG_HOME"] or HOME / ".config")
```

Comment:
Writing like this can cause a KeyError. Use `get` method on dictionaries instead :)